packages/oasis-actions/src/helpers/swap/getSwapData.ts

```diff
@@ -0,0 +1,61 @@
+import BigNumber from 'bignumber.js'
+
+import { SwapData } from '../../types'
+import { acceptedFeeToken } from './acceptedFeeToken'
+import { calculatePreSwapFeeAmount } from './calculatePreSwapAmount'
+
+type GetSwapDataArgs<Tokens> = {
+  swapAmountBeforeFees: BigNumber
+  debtToken: { symbol: Tokens; precision?: number }
+  collateralToken: { symbol: Tokens; precision?: number }
+  slippage: BigNumber
+}
+
+export async function getSwapDataHelper<Addresses, Tokens>({
+  fromTokenIsDebt,
```

Comment:
What if we want to swap the deposit token for the position token? 
I have dai and want to payback my debt in `USDC`.
SwapHelper shouldn't be aware of concept collateral or debt. Swap is an abstract concept.